### DISCUSSION TITLE
grpc support for espressif (esp32) platforms.

### DISCUSSION THREAD

**Discussion by mmmunir966** [actions/virtual-environments#5777] [General questions]
Hi, 
I am looking grpc c++ support for espressif devices. 
Espressif provides their microcontrollers and support C and C++ build system using esp-idf platform. I am able to build the protobuf in that platform, but receiving errors when add grpc file generations. The files are generated, but then receive an error of platform not found. 
I am using MacOs 12.2 with chipset Apple M1 Pro and making cross compilation for esp-idf platforms. 

Any help would be much appreciated.

**Comment by JDoGG1** [comment]
Did you ever find a solution to this
